fury/tests/test_primitive.py

```diff
@@ -2,18 +2,24 @@
 import numpy.testing as npt
 import fury.primitive as fp
 
-
 def test_vertices_primitives():
-    l_primitives = [(fp.prim_square, (4, 3)),
-                    (fp.prim_box, (8, 3))]
+    l_primitives = [(fp.prim_square, (4, 3), -.5, .5, 0),
+                    (fp.prim_box, (8, 3), -.5, .5, 0),
+                    (fp.prim_star, (10, 3), -3, 3, -0.06666666666666667),
+                    (fp.prim_rhombicuboctahedron, (24, 3), -4, 4, 0)]
 
-    for func, shape in l_primitives:
+    for func, shape, min, max, mean in l_primitives:
```

Comment:
`min, max, mean` are predefined by python so it is better to avoid this variable name. Can you change them? Just put `e_min, ...`. the e is for expected